modules/Story/Share/Share.tsx

```diff
@@ -135,7 +135,7 @@ export function Share({
                                     successMessage="Copied to clipboard"
                                     onClick={handleCopyText}
                                 >
-                                    Copy story text
+                                    Copy text
```

Comment:
I've noticed the translations are missing completely for the new functionality. @yuriyyakym @pestaa 

<img width="779" alt="Screenshot 2025-02-24 at 11 56 18" src="https://github.com/user-attachments/assets/7e8f21e9-85d9-4d2b-b662-1ceff27dce9a" />

Reply:
Yes, it was mentioned in one of project updates. I need assistance with this